articles/logic-apps/workflow-definition-language-functions-reference.md

```diff
@@ -2195,7 +2195,7 @@ guid('<format>')
 
 | Parameter | Required | Type | Description |
 | --------- | -------- | ---- | ----------- |
-| <*format*> | No | String | A single [format specifier](https://msdn.microsoft.com/library/97af8hh4) for the returned GUID. By default, the format is "D", but you can use "N", "D", "B", "P", or "X". |
+| <*format*> | No | String | A single [format specifier](/dotnet/api/system.guid.tostring.7.2#System_Guid_ToString_System_String_) for the returned GUID. By default, the format is "D", but you can use "N", "D", "B", "P", or "X". |
```

Comment:
Please use this URL:

https://docs.microsoft.com/dotnet/api/system.guid.tostring?redirectedfrom=MSDN&view=netframework-4.7.2#System_Guid_ToString_System_String_

Reply:
Should be https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring#System_Guid_ToString_System_String

Reply:
Should be https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring#System_Guid_ToString_System_String_